packages/resolver-hoogle-search/index.js

```diff
@@ -3,7 +3,7 @@ function handleResponse(hoogleSearchUrl, target, response) {
     return item.url.includes(`docs/${target.replace(/\./g, '-')}.html`);
   };
 
-  if (response.length > 0) {
+  if (response?.length > 0) {
```

Comment:
This was causing an unresolved promise error in the `calls the hoogle search api` test since it wasn't setting the json value to anything.